build.gradle

```diff
@@ -90,6 +112,13 @@ sonarqube {
 }
 
 test {
+  useJUnitPlatform()
```

Comment:
This stuff was copied wholesale from EsrReconciliationService, but it needs to be either implemented properly or removed.